examples/image_classifier/streaming_video/README.md

```diff
@@ -0,0 +1,147 @@
+Install OpenCV with the following command
+```
+pip install opencv-python
+```
+
+To make use of TorchServe's dynamic batching feature, we need to send asynchronous http requests. Hence, we are using [requests-futures](https://github.com/ross/requests-futures) in this example.
+Install requests-futures with the following command
+```
+pip install requests-future
+```
+
+## With TorchServe batching
+
+On the client side, we have one thread for reading frames from a video source and another thread which sends the read frames as http request to TorchServe for image classification inference. We are using an asynchronous http requests as we want to make use of TorchServe batching.
+We send one frame in each request and let TorchServe handle the batching
+TorchServe is setup to process batch size of 4 in this example.
+TorchServe receives individual requests, batches the requests to make a single inference request and sends out individual responses to the requests received.
+
+
+### Create a resnet-18 eager mode model archive, register it on TorchServe and run inference on a streaming video
+
+Run the commands given in following steps from the parent directory of the root of the repository. For example, if you cloned the repository into `/home/my_path/serve`, run the steps from `/home/my_path`
+
+```bash
+python examples/image_classifier/streaming_video/create_mar_file.py
+
+torchserve --start --model-store model_store --models resnet-18=resnet-18.mar --ts-config examples/image_classifier/streaming_video/config.properties
+
+python examples/image_classifier/streaming_video/request.py
+```
+
+The default batch size is 4.
+On the client side, we should see the following output
+
+```bash
+With Batch Size 4, FPS at frame number 20 is 24.7
+{
+  "tabby": 0.5186409950256348,
+  "tiger_cat": 0.29040342569351196,
+  "Egyptian_cat": 0.10797449946403503,
+  "lynx": 0.01395314373075962,
+  "bucket": 0.006002397276461124
+}
+{
+  "tabby": 0.5186409950256348,
+  "tiger_cat": 0.29040342569351196,
+  "Egyptian_cat": 0.10797449946403503,
+  "lynx": 0.01395314373075962,
+  "bucket": 0.006002397276461124
+}
+{
+  "tabby": 0.5186409950256348,
+  "tiger_cat": 0.29040342569351196,
+  "Egyptian_cat": 0.10797449946403503,
+  "lynx": 0.01395314373075962,
+  "bucket": 0.006002397276461124
+}
+{
+  "tabby": 0.5186409950256348,
+  "tiger_cat": 0.29040342569351196,
+  "Egyptian_cat": 0.10797449946403503,
+  "lynx": 0.01395314373075962,
+  "bucket": 0.006002397276461124
+}
+```
+
+## With Client-Side batching
+
+On the client side, we have one thread for reading frames from a video source and another thread which batches(size n) the read frames and sends the request to TorchServe for image classification inference.
+To send the batched data, we create a json payload of n frames.
+On the TorchServe side, we read the json payload and preprocess the n frames. The postprocess function in the handler returns the output as a list of length 1.
```

Comment:
the number of frames = the client side batch size. is my understanding correct?

Reply:
In this example, number of frames = 350+ , batch_size = 4